Don't require python3 in PATH

Once packaged, the arcanoid.py script has an absolute path shebang to
python3, so there is no need to also require python3 in PATH.

This prevents leaking in a python3 in PATH in the environment which arc
runs linters etc.

Author: bjorn.forsman@gmail.com
diff -uNr arcanist.orig/src/workflow/ArcanistAnoidWorkflow.php arcanist.new/src/workflow/ArcanistAnoidWorkflow.php
--- arcanist.orig/src/workflow/ArcanistAnoidWorkflow.php	2022-03-31 13:23:30.865095192 +0200
+++ arcanist.new/src/workflow/ArcanistAnoidWorkflow.php	2022-04-01 12:19:15.644159639 +0200
@@ -24,13 +24,6 @@
   }
 
   public function runWorkflow() {
-    if (!Filesystem::binaryExists('python3')) {
-      throw new PhutilArgumentUsageException(
-        pht(
-          'The "arc anoid" workflow requires "python3" to be available '.
-          'in your $PATH.'));
-    }
-
     $support_dir = phutil_get_library_root('arcanist');
     $support_dir = dirname($support_dir);
     $support_dir = $support_dir.'/support/';
